htdocs/install/mysql/migration/13.0.0-14.0.0.sql

```diff
@@ -373,6 +373,15 @@ UPDATE llx_propal SET date_signature = date_cloture WHERE date_signature IS NULL
 
 
 ALTER TABLE llx_product ADD COLUMN batch_mask VARCHAR(32) NULL;
+ALTER TABLE llx_product ADD COLUMN life_duration VARCHAR(6);
+ALTER TABLE llx_product ADD COLUMN control_periodicity VARCHAR(6);
+
+ALTER TABLE llx_product_lot ADD COLUMN eol_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN manufacturing_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN scrapping_date datetime;
+ALTER TABLE llx_product_lot ADD COLUMN commissionning_date datetime;
```

Comment:
Same question. What is goal of this date ? What is translation in french ?

Reply:
commisionning = mise en service

Reply:
For which use cas do you need this date. What do you want to store in this date ?

Reply:
There are many possibilities. We could offer them in the settings:
-Date of purchase
-Date of receipt (from buying or manufacturing)
-Shipping date
Or leave the field free

Reply:
Initially we can do it in mass then by product in a second time

Reply:
Having such a date on the lot is not relevant.
If you buy 10 of Lot A the 1st and 10 of Lot A the 2nd, you can't store a date into the table that store the unique lot A. 
Suggested date are information related to a stock movement, or a sale, or a shipment, but is not not intrasec of a lot. I remove it for the moment so we can validate other fields.

Reply:
I agree